doc/tutorials/pick_and_place_with_moveit_task_constructor/launch/mtc_demo.launch.py

```diff
@@ -0,0 +1,110 @@
+import os
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess
+from launch_ros.actions import Node
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+
+
+def generate_launch_description():
```

Comment:
Is this a direct copy of `panda_moveit_config demo.launch.py`? If so, let's delete it and just use that one.

If it's just a bit different, then you could call that launch file from this one. 

https://answers.ros.org/question/363687/ros2-call-launch-file-from-another-launch-file/

Reply:
This is an altered version of [moveit_task_constructor_demo demo.launch.py](https://github.com/ros-planning/moveit_task_constructor/blob/ros2/demo/launch/demo.launch.py)

There are definitely some similarities between the two, but it seems the panda_moveit_config launchfile will launch RViz with its own RViz custom config, so if I were to call that launch file from this one, I don't think I can launch RViz with the mtc.rviz config in this package.

Reply:
ok, thanks for checking